src/Servers/Kestrel/samples/Http2SampleApp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/aspnetcore:2.0.0-stretch
+FROM mcr.microsoft.com/dotnet/aspnet:2.1-stretch-slim
```

Comment:
Other alternatives:

1. use `mcr.microsoft.com/dotnet/aspnet:3.1-buster-slim` tag (which is Docker 10)
2. backport c5c8f4385ee8